crates/revm/src/handler.rs

```diff
@@ -77,19 +77,19 @@ impl<'a, EXT, DB: Database> EvmHandler<'a, EXT, DB> {
 
     /// Handler for optimism
     #[cfg(feature = "optimism")]
-    pub fn optimism<SPEC: Spec>() -> Self {
-        let mut handler = Self::mainnet::<SPEC>();
+    pub fn optimism<SPEC: Spec>(with_reward_beneficiary: bool) -> Self {
+        let mut handler = Self::mainnet::<SPEC>(with_reward_beneficiary);
         handler.cfg.is_optimism = true;
-        handler.append_handler_register(HandleRegisters::Plain(
```

Comment:
Why the change in signature and going from `Plain` to `Box`?

Reply:
I reverted the signature change in https://github.com/risechain/revm/pull/2